jitsi_meet_wrapper/pubspec.yaml

```diff
@@ -1,6 +1,7 @@
 name: jitsi_meet_wrapper
 description: A Flutter wrapper around the official JitsiMeetSDKs. With this package, you can launch Jitsi meetings in a native view.
 version: 0.0.5
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
```

Comment:
Are you sure that this line is correct? Because this package is published to pub.dev.

Reply:
Fixed in latest commit @nilsreichardt